src/microbe_stage/PlayerMicrobeInput.cs

```diff
@@ -163,6 +163,26 @@ public void ToggleEngulf()
         }
     }
 
+    [RunOnKeyDown("g_eject_engulfed")]
+    public void EjectAllEngulfed()
+    {
+        ref var engulfer = ref stage.Player.Get<Engulfer>();
+
+        if (engulfer.EngulfedObjects == null || engulfer.EngulfedObjects?.Count == 0)
+            return;
+
+        if (!stage.HasAlivePlayer)
```

Comment:
This check should be *before* the engulfer component is retrieved, otherwise the game might crash if the player object has been destroyed and no longer has an engulfer component.